### PR TITLE
Added command to create /etc/apt/keyrings

### PR DESCRIPTION
**Pull request by deepeshramnani**
For releases older than Debian 12 and Ubuntu 22.04, the GCP public signed key step ends with the following error: 
```
instance-3:~$ sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg 
curl: (23) Failed writing body (0 != 1210)
```

This is because the /etc/apt/keyrings directory does not exist by default, as mentioned in the note at the bottom of this section. Hence, adding `mkdir -p` command to create it.

I have seen people in multiple forums confused with the above error, which does not indicate that the /etc/apt/keyrings directory is missing.

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
 2. Download the Google Cloud public signing key:
 
    ```shell
+   sudo mkdir -p /etc/apt/keyrings/
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:133]:
```suggestion
  # /etc/apt/keyrings might not yet exist on older operating systems.
  # On the latest Debian or Ubuntu releases, you can skip the next 3 lines.
  [ ! -d /etc/apt/keyrings ] && \
    sudo mkdir -p /etc/apt/keyrings && \
    sudo chmod 0755 /etc/apt/keyrings
```

**Review by sftim — COMMENTED** [review]
If we add this, let's drop the note:
> In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
> You can create this directory if you need to, making it world-readable but writeable only by admins.

*and* we should also make sure that the permissions on that directory are correct. The system can break (degrade) for non-admin users if we don't make it world readable.

**Comment by deepeshramnani** [comment]
@sftim - Thanks, committed suggested changes and dropped the note.

**Comment by dipesh-rawat** [comment]
@deepeshramnani  We do occasionally see Netlify build failures due to timeouts. Could you try to trigger the build again using `git commit --amend --no-edit && git push --force-with-lease`

**Comment by sftim** [comment]
@deepeshramnani please do rebase this against _main_

**Comment by deepeshramnani** [comment]
> @deepeshramnani please do rebase this against _main_

@sftim - Done 👍🏻

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -157,6 +157,11 @@ The following methods exist for installing kubectl on Linux:
 2. Download the Google Cloud public signing key:
 
    ```shell
+   # /etc/apt/keyrings might not yet exist on older operating systems.
+   # On the latest Debian or Ubuntu releases, you can skip the next 3 lines.
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:161]: “the latest” isn't quite right. Try
```suggestion
   # Older releases of Debian (version 11 or older) and Ubuntu (21.10 or older) do not provide
   # that directory as part of the operating system.
   # If you are running a current enough release of Debian or Ubuntu, you can skip the next 3 lines.
```

**Comment by sftim** [comment]
I'm in two minds about making this change. What do other reviewers think?

**Comment by divya-mohan0209** [comment]
IMO, I don't think we need this change given the current note that exists.

> In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default, and can be created using `sudo mkdir -m 755 /etc/apt/keyrings`

**Comment by reylejano** [comment]
Hi @deepeshramnani , I'm closing this PR in agreement with another reviewer that the change proposed in this PR is not needed and this PR has a merge conflict and requires rebase
If you have time to rebase and feel otherwise, feel free to reopen this PR with `/reopen` in a comment
/close

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/pull/43626

**Pull request closed without merging**